mobile/cmp/grid/impl/ColChooser.ts

```diff
@@ -60,7 +60,6 @@ export const [ColChooser, colChooser] = hoistCmp.withFactory<ColChooserProps>({
                     onDragEnd: impl.onDragEnd,
                     items: [
                         panel({
-                            title: 'Visible Columns',
```

Comment:
This did not seem to add much and stacked another layer of headers for mobile colChooser.